core/src/test/JSONConnector/JSONConnector.ts

```diff
@@ -51,11 +51,10 @@ export class JSONConnector extends pcf.PConnector {
           "ExtPhysicalElement",
           "ExtPhysicalType",
           "ExtSpace",
-          "ExtSpatialCategory", "ExtSpatialSubcategory",
+          "ExtSpatialCategory", "SpatialSubcategory",
```

Comment:
Why do we need both ExtSpatialSubcategory and SpatialSubcategory?

Reply:
The first is a spatial category. I think Zach's `Ext--` prefix (*ext*ension?) refers to the dynamic `TestSchema`. `SpatialSubcategory` is a subcategory. I took off the `Ext--` because it doesn't use a class from the dynamic schema.

I called it `SpatialSubcategory` rather confusingly because the DMO for it can only be used to attach subcategories to spatial categories. This is because PCF relies on the DMO attached to the subcategory's parent to find its ID. It needs to know what IR entity it came from to construct its code value.

Reply:
@jackson-at-bentley can you please put a comment to this effect in the source?

Reply:
Done!